cmd/server.go

```diff
@@ -25,9 +34,42 @@ func startServer(c *grumble.Context) error {
 }
 
 func stopServer(c *grumble.Context) error {
-	panic("implement me")
+	if len(c.Args) == 0 {
+		if db == nil {
+			fmt.Println("no server to stop")
+			return nil
+		}
+		err := db.Close()
+		if err != nil {
+			fmt.Println("flydb stop error: ", err)
+			return err
+		}
+		fmt.Println("flydb stop success")
+		db = nil
+	}
+	return nil
 }
 
 func cleanServer(c *grumble.Context) error {
-	panic("implement me")
+
+	if len(c.Args) == 0 {
+		if db == nil {
+			err := errors.New("no server to clean")
+			fmt.Println("flydb clean error: ", err)
+			return err
+		}
+		err := db.Close()
+		if err != nil {
+			fmt.Println("flydb clean error: ", err)
+			return err
+		}
+		err = os.RemoveAll(options.DirPath)
```

Comment:
In addition to this, you also need to clean the history file. You can check for any history files that appear in your tmp folder during the startup process of cli